.github/workflows/release_macOS.yml

```diff
@@ -83,6 +48,6 @@ jobs:
         run: echo "VITE_LD_ENVIRONMENT_ID=${{ secrets.VITE_LD_ENVIRONMENT_ID }}" > .env.production
 
       - name: Build artifacts.
-        run: yarn release:mac:arm64
+        run: yarn release:mac
```

Comment:
I don't think we can compile intel builds on this arm runner `macos-latest-xlarge` because of the bytecode issue https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/pull/574

Reply:
I think we should try since it is being 5 months since the change.
I will modify the other file to see how it will work.